pkg/config/config.go

```diff
@@ -34,3 +41,19 @@ const (
 
 	KourierIngressClassName = "kourier.ingress.networking.knative.dev"
 )
+
+// ServiceDomains returns the external and internal service's respective domain.
+//
+// Example: kourier.kourier-system.svc.cluster.local.
+func ServiceDomains() (string, string) {
+	suffix := "." + GatewayNamespace() + ".svc." + network.GetClusterDomainName()
+	return ExternalServiceName + suffix, InternalServiceName + suffix
+}
```

Comment:
can we use https://github.com/knative/pkg/blob/c46a18644992a3ead940f96f7b9e9078c0dcbb28/network/domain.go#L39

Reply:
Thanks, yes we can. Pushed!